packages/ui/src/components/Slider/components/SingleSlider.tsx

```diff
@@ -124,24 +120,21 @@ export const SingleSlider = ({
     return []
   }, [options])
 
-  const onChangeCallback = useCallback(
-    (newValue: number) => {
-      setValueToShow(options ? options[newValue].value : newValue)
-      setComputedValue(newValue ?? min)
-      onChange?.(newValue ?? min)
-    },
-    [min, onChange, options],
-  )
+  const internalOnChangeRef = useRef((newValue: number) => {
```

Comment:
Do we need the useRef ?

Reply:
Yes because we use it in useEffect and eslint rule doesnt allow us to omit it in deps array